install/installer/pkg/components/gitpod/cronjob.go

```diff
@@ -63,6 +63,10 @@ func cronjob(ctx *common.RenderContext) ([]runtime.Object, error) {
 												Name:  "GITPOD_INSTALLATION_VERSION",
 												Value: ctx.VersionManifest.Version,
 											},
+											{
+												Name:  "GITPOD_INSTALLATION_PLATFORM",
+												Value: "installer",
```

Comment:
Similar to Simon's comment, I would propose to set `unknown` here as well instead of `installer` (which is not a platform in the strict sense). WDYT?